spec/javascripts/components/story/expanded_story/expanded_story_controls_spec.js

```diff
@@ -55,76 +55,125 @@ describe('<ExpandedStoryControls />', () => {
     });
   });
 
-  describe('when the user click on cancell', () => {
+  describe('when the user click on cancel', () => {
```

Comment:
Just a typo: when the user **clicks** on cancel

Reply:
good catch!